src/containers/App.jsx

```diff
@@ -40,6 +43,33 @@ class App extends Component {
     }
   }
 
+  onFabClick = (event) => {
+    if (event.type === 'keydown' && event.keyCode !== 13) {
```

Comment:
What does this do?

Reply:
It makes the modal to be opened by hitting the ENTER key when focus is on the FAB.

Reply:
Seems not to be working at the moment

Reply:
How did you test it @Farhanisto ?

Reply:
I fetched your branch and ran it... when the mouse  is on the component and i press enter the model is not revealed.

Reply:
@Farhanisto use the Tab key to navigate a component then press enter.

Reply:
Cool it works

Reply:
👍